lib/templatesimplifier.cpp

```diff
@@ -750,6 +763,17 @@ static void getFunctionArguments(const Token *nameToken, std::vector<const Token
         args.push_back(argToken);
 }
 
+static bool isConstMethod(const Token* nameToken)
+{
+    const Token* functionToken = getFunctionToken(nameToken);
+    if (!functionToken)
+        return false;
+    const Token* endToken = functionToken->link();
+    if (!endToken)
```

Comment:
this check is technically redundant as simpleMatch would return false also for a null pointer.